core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -5004,7 +5004,7 @@ protected function listView()
     ';
 
 				$colspan = 1;
-				$label = ($blnDraft ? '<p class="draft-label">' . $GLOBALS['TL_LANG']['MSC']['draft'] . '</p> ' : '') . $label;
+				$label = $blnDraft ? '<p class="draft-label">' . $GLOBALS['TL_LANG']['MSC']['draft'] . '</p> ' : $label;
```

Comment:
This fix is unnecessary, because `$label` is never an array here. There is a `preg_replace()` on line 4975 that would already fail if `$label` was an array.

Reply:
`preg_replace` allows for an array argument!